eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -95,8 +95,6 @@ def fitJWST(eventlabel, s4_meta=None):
             
             t0 = time.time()
             
-            meta = load_specific_s4_meta_info(old_meta, run_i, spec_hw_val, bg_hw_val)
```

Comment:
Why was this line deleted? It is a very important line

Reply:
There was a github conflict and I think I messed up when fixing it! Should be resolved now